test/charm_tests.lua

```diff
@@ -918,3 +918,26 @@ if next(SMODS.find_mod 'Cryptid') then
     }
     --#endregion
 end
+
+if next(SMODS.find_mod 'GARBPACK') then -- Garbshit
+    --#region Virus
+    Balatest.TestPlay {
+        name = 'virus',
+        category = { 'charms', 'virus', 'garbshit' },
+
+        deck = { cards = {
+            { r = '5', s = 'D' },
+            { r = 'A', s = 'S' }
+        } },
+        execute = function()
+            equip 'BakeryCharm_Bakery_Virus'
+            Balatest.play_hand { '5D' }
+            Balatest.wait_for_input()
+        end,
+        assert = function()
+            Balatest.assert_eq(#G.hand.cards, 1)
+            Balatest.assert_eq(G.hand.cards[1].ability.name, 'm_garb_infected')
+        end
+    }
```

Comment:
The following cases are untested:
- Every card in hand is already infected
- Every card in hand is already enhanced but not infected
- There are 0 cards left in hand (either hand size <= 5 or the blind was defeated with that hand)

Reply:
Resolved in b4a1570